example/debug/odp_debug.c

```diff
@@ -382,12 +383,19 @@ static int timer_debug(void)
 	if (timer_capa.max_tmo.max_tmo < max_tmo)
 		max_tmo = timer_capa.max_tmo.max_tmo;
 
-	if (timer_capa.max_tmo.res_ns > res)
-		res = timer_capa.max_tmo.res_ns;
+	memset(&timer_res_capa, 0, sizeof(odp_timer_res_capability_t));
+	timer_res_capa.max_tmo = max_tmo;
+	if (odp_timer_res_capability(ODP_CLOCK_CPU, &timer_res_capa)) {
+		ODPH_ERR("Timer resolution capability failed\n");
+		return -1;
+	}
+
+	if (timer_res_capa.res_ns < res)
+		res = timer_res_capa.res_ns;
```

Comment:
Requested resolution is too high, if it is smaller than capa.res_ns. So, the comparison should be other way around

```
if (timer_res_capa.res_ns > res)
		res = timer_res_capa.res_ns;
```